pkg/transform/oauth/github_test.go

```diff
@@ -39,17 +39,18 @@ func TestTransformMasterConfigGithub(t *testing.T) {
 
 		identityProviders = append(identityProviders,
 			oauth.IdentityProvider{
-				provider.Kind,
-				provider.APIVersion,
-				identityProvider.MappingMethod,
-				identityProvider.Name,
-				identityProvider.Provider,
-				provider.File,
-				nil,
-				nil,
-				nil,
-				identityProvider.UseAsChallenger,
-				identityProvider.UseAsLogin,
+				Kind:            provider.Kind,
+				APIVersion:      provider.APIVersion,
+				MappingMethod:   identityProvider.MappingMethod,
+				Name:            identityProvider.Name,
+				Provider:        identityProvider.Provider,
+				HTFileName:      provider.File,
+				HTFileData:      nil,
```

Comment:
If these are pointer types, they will naturally default to nil, so you can safely just leave them out of the initializer.

Reply:
True. Now that it's not in the format it was these can go away. I'll remove them.